internal/pkg/nats/options.go

```diff
@@ -58,6 +58,8 @@ type ClientOptions struct {
 	QueueGroup              string
 	Deliver                 string
 	DefaultPubRetryAttempts int
+	NKeySeedFile            string
+	CredentialsFile         string
```

Comment:
How do these differ from 
```
KeyFile = ""
CertFile = ""
```
Used for EdgeX

Reply:
NATS has its own TLS facilities that we attempt to support using those properties.  NKeys is a NATS-specific public/private key scheme for authentication, and the Credentials file is a means to use a JWT layer on top of it.  I think both could end up working nicely with the secret provider eventually but focused on these file-based options first since the NATS client takes care to wipe keys from memory and I'd like to avoid propagating the build flag usage too widely.